shared/css/customized.css

```diff
@@ -0,0 +1,5 @@
+/* place project specific CSS customization here */
+
+.reveal img.title-logo {
+  border: 0;
```

Comment:
```suggestion
  border: 0;
  box-shadow: unset;
```

this would remove the show - like in this screenshot
![image](https://user-images.githubusercontent.com/15704517/143266065-323f5ab9-d21c-411f-a178-fc49f6ac51fb.png)